README.md

```diff
@@ -93,7 +93,112 @@ if __name__ == '__main__':
   asyncio.run(main())
 ```
 
-Once connected, the entire functionalities offered by Memphis are available.
+```python
+    async def connect(
+        self,
+        host: str,
+        username: str,
+        account_id: int = 1, # Cloud use only, ignored otherwise
+        connection_token: str = "", # JWT token given when creating client accounts
+        password: str = "", # For password-based connections
+        port: int = 6666,
+        reconnect: bool = True,
+        max_reconnect: int = 10,
+        reconnect_interval_ms: int = 1500,
+        timeout_ms: int = 2000,
+        # For TLS connections: 
+        cert_file: str = "", 
+        key_file: str = "",
+        ca_file: str = "",
+    )
+```
+
+The connect function in the Memphis class allows for the connection to Memphis. Connecting to Memphis (cloud or open-source) will be needed in order to use any of the other functionality of the Memphis class. Upon connection, all of Memphis' features are available.
+
+What arguments are used with the Memphis.connect function change depending on the type of connection being made.
+
+For details on deploying memphis open-source with different types of connections see the [docs](https://docs.memphis.dev/memphis/memphis-broker/concepts/security).
+ 
+The default connection type (for cloud) is using a JWT token-based connection. Here is an example of using memphis.connect with a JWT token:
```

Comment:
@idanasulinmemphis @shay23b not sure I understand this comment. Also, the default connection method is password-based.

Reply:
@John-Memphis I dont see any example with `account id` for connecting with the Cloud

Reply:
@yanivbh1 I'll add an example for that.
![image](https://github.com/memphisdev/memphis.py/assets/136013599/266e97b3-d2da-4959-91ae-166b3bd60988)
![image](https://github.com/memphisdev/memphis.py/assets/136013599/a34e67ce-61f0-4a47-ad04-e143596182ff)

All of the repos say that the default connection is token based. I've only been using Memphis with open-source and deployed with docker--where password is the default. I was assuming it was a kubernetes thing or a memphis cloud thing that used tokens by default.

If that's not the case I'll go and change that in all of the repos.